package/pyproject.toml

```diff
@@ -86,6 +86,7 @@ analysis = [
     "seaborn",
     "scikit-learn",
     "tidynamics>=1.0.0",
+    "dask"
```

Comment:
This is under `analysis`. It needs to be move under `parallel`, to be consistent with `setup.py`.